docs/OV_Runtime_UG/automatic_batching.md

```diff
@@ -68,10 +70,10 @@ This _automatic batch size selection_ assumes that the application queries the `
 
 @endsphinxtabset
 
-If not enough inputs were collected, the `timeout` value makes the transparent execution fall back to the execution of individual requests. Configuration-wise, this is the AUTO_BATCH_TIMEOUT property.
-The timeout, which adds itself to the execution time of the requests, heavily penalizes the performance. To avoid this, in cases when your parallel slack is bounded, give the OpenVINO an additional hint.
+If not enough inputs were collected, the `timeout` value makes the transparent execution fall back to the execution of individual requests. Configuration-wise, this is the `AUTO_BATCH_TIMEOUT` property.
```

Comment:
```suggestion
If not enough inputs were collected, the `timeout` value makes the transparent execution fall back to the execution of individual requests. This can be configured via the `AUTO_BATCH_TIMEOUT` property.
```